ComposeSamples/README.md

```diff
@@ -18,6 +18,8 @@ To learn how to load apps on the Surface Duo emulator, see the [documentation](h
 
 ## Prerequisites
 
+- [Android Studio Canary](https://developer.android.com/studio/preview)
```

Comment:
We don't need to state `Android Studio Canary` here, since it is mandatory for Jetpack Compose development.

Reply:
ok